README.md

```diff
@@ -72,44 +72,47 @@ The complete flow is as follows:
 Calcite reads the schema information from storage, and the Executor stores a reference to Data Manager for buffer/storage access during a query. 
 
 The return from RelAlgExecutor is a ResultSet object which can be converted to Arrow and to pandas:
-```
+```python
      df = res.to_arrow().to_pandas()
 ```
 
 ## Examples
 
 Standalone examples are available in the `examples` directory. Most examples run via Jupyter notebooks. 
 
+
 ## Build
 
-HDK includes components from external libraries as submodules. To clone the project, either use `git clone --recurse-submodules` to initially clone the repo, or clone without using flags and then run:
+### Dependencies 
 
-```
-git submodule init
-git submodule update
+Miniconda installation required. (Anaconda may produce build issues.) Use one of this [miniconda installers](https://docs.conda.io/en/latest/miniconda.html).
+
+Conda environments are used for HDK development. Use the yaml file in `omniscidb/scripts/`:
+
+```bash
+     conda env create -f omniscidb/scripts/mapd-deps-conda-dev-env.yml
+     conda activate omnisci-dev
 ```
 
-When pulling changes from upstream, be sure to update submodule references using `git submodule update`.
+### Compilation
 
 If using a Conda enviornment, run the following to build and install HDK:
 
-```
-mkdir build
-cd build
-cmake -DENABLE_CONDA=on ..
-make -j
-make install
+```bash
+    mkdir build && cd build
+    cmake -DENABLE_CUDA=off ..
```

Comment:
`Heterogeneous` is a part of our library name but build instructions are for the CPU-only version. Should we propose CUDA enabled build as the default option and describe CPU-only version as an alternative for cases when the CUDA toolkit is not available?